Assets/Scripts/Controllers/SoundController.cs

```diff
@@ -56,8 +56,11 @@ private void OnTileChanged(Tile tileData)
             return;
         }
 
-        AudioClip ac = Resources.Load<AudioClip>("Sounds/Floor_OnCreated");
-        AudioSource.PlayClipAtPoint(ac, Camera.main.transform.position);
-        soundCooldown = 0.1f;
+        if (tileData.walkCount == 0 && tileData.damageEventCount == 0)
```

Comment:
We could add an alternative sound for walking on tiles, but it didn't seem like we should be playing the floor tile created sound every time a tile is walked on.